.github/workflows/actions.yml

```
@@ -45,6 +45,27 @@ jobs:
           - name: zip up the LCP
             run: |
               zip -r "kais-npc-rebakes-deployables-${{ github.event.release.tag_name }}.lcp" *.json
+          - name: upload LCP to release
+            env:
+              GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            run: |
+              gh release upload --clobber ${{ github.event.release.tag_name }} *.lcp
+        bundle-wallflower-lcp:
```

```suggestion
    bundle-wallflower-lcp:
```
yaml files are whitespace sensitive. make sure this line is of equal indentation than `bundle-lcp` and `bundle-deployables-lcp`